packages/react/src/Logo/Logo.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import LogoAmsterdam from '@amsterdam/design-system-assets/logo/amsterdam.svg'
+import LogoGGD from '@amsterdam/design-system-assets/logo/ggd-amsterdam.svg'
+import LogoStadsarchief from '@amsterdam/design-system-assets/logo/stadsarchief.svg'
+import LogoStadsbankLening from '@amsterdam/design-system-assets/logo/stadsbank-lening.svg'
+import LogoVGA from '@amsterdam/design-system-assets/logo/vga-verzekeringen.svg'
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes } from 'react'
+
+export interface LogoProps extends HTMLAttributes<HTMLElement> {
+  variant?: 'amsterdam' | 'ggd-amsterdam' | 'stadsarchief' | 'stadsbank-lening' | 'vga-verzekeringen'
+}
+
+function getLogoVariant(variant: LogoProps['variant']) {
```

Comment:
I think we can simplify this. The switch in a function can be replaced with a TypeScript `Record` like so:

```ts
type LogoBrand = 'amsterdam' | 'ggd-amsterdam' | 'stadsarchief' | 'stadsbank-van-lening' | 'vga-verzekeringen'

export interface LogoProps extends HTMLAttributes<HTMLElement> {
  variant?: LogoBrand
}

const logoConfig: Record<LogoBrand, ReactNode> = {
  amsterdam: LogoGGD,
  'ggd-amsterdam': LogoStadsarchief,
  stadsarchief: LogoStadsbankLening,
  'stadsbank-van-lening': LogoVGA,
  'vga-verzekeringen': LogoAmsterdam,
}
```

(If we need more values later we can extend to `{ component: ReactNode, key: value }` objects.)

This allows for a smaller render function:

```ts
const LogoComponent = logoConfig[brand]

return (
  <span {...restProps} ref={ref} className={clsx('amsterdam-logo', className)}>
    {LogoComponent}
  </span>
)
```

This renders the `svg` inline rather than in an image, which I think we should do. We can store the description in the image itself through the `title` element.

[This Smashing article](https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/) looks like a good resource on this. [MDN here](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_In_HTML_Introduction).

Reply:
This became kind of a struggle with typescript/rollup/react not importing svg as a ReactNode (or SVGElement) and required a lot of workarounds and extra plugins for the linters. So I changed it to the way Icon works and built components for the svg's (with svgomg optimized svgs this time) and it works without jumping hoops. 

I've used the Smashing article to add a11y to the svg, thanks, good article.